src/modules/services/postgres.nix

```diff
@@ -276,6 +281,19 @@ in
         CREATE ROLE bar;
       '';
     };
+
+    customPgHbaFile = lib.mkOption {
```

Comment:
I'd name it `hbaConf` and make it accept a string, for consistency with other options.

Reply:
Your right! Done.